packages/diff-so-fancy/build.sh

```diff
@@ -0,0 +1,24 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/so-fancy/diff-so-fancy"
+TERMUX_PKG_DESCRIPTION="Good-lookin' diffs. Actually… nah… The best-lookin' diffs"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_VERSION="1.4.4"
+TERMUX_PKG_SRCURL="${TERMUX_PKG_HOMEPAGE}/archive/v${TERMUX_PKG_VERSION}.tar.gz"
```

Comment:
We don't use the value of `TERMUX_PACKAGE_HOMEPAGE` as part of the SRCURL.
This would have to be changed if the project gets a "proper" website.
```suggestion
TERMUX_PKG_SRCURL="https://github.com/so-fancy/diff-so-fancy/archive/v${TERMUX_PKG_VERSION}.tar.gz"
```

Reply:
https://github.com/termux/termux-packages/blob/45c05023e443bbe8f8df06281c0fccc9e9355260/packages/zsh-completions/build.sh#L7

Do I change both or just mine?

Reply:
Changed just mine in 3f36bc80c , let me know if you wish me to touch the other package in a third commit.

Reply:
Just what's in this PR.
If you or anyone else wants to do a followup PR to fix that particular bit of overlooked policy drift that can be done separately.